### PR TITLE
Convert stories to composition API and fix broken ones

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Related to #1135 by @dhruvkb 
Related to #1220 by @dhruvkb

## Description

The Storybook has been a bit neglected lately. This PR updates the stories to use the composition API (which surprisingly works and fixes the broken stories). There are many more stories but in the interest of keeping the PR reviewable, this will be accomplished in several parts.

## Testing Instructions

Run the [production Storybook](https://wordpress.github.io/openverse-frontend) to find the broken stories. Run the Storybook from this PR to see that they've been fixed.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/VNoticeBar/meta/NoticeBar.stories.mdx [hunk 1]
```diff
@@ -10,10 +10,12 @@ import NoticeBar from '~/components/VNoticeBar/VNoticeBar'
 
 <Meta title="Components/Notice bar" component={NoticeBar} />
 
-export const Template = (args, { argTypes }) => ({
-  template: `<NoticeBar>{{ text }}</NoticeBar>`,
+export const Template = (args) => ({
+  template: `<NoticeBar>{{ args.text }}</NoticeBar>`,
   components: { NoticeBar },
```
Comment (obulat) [inline src/components/VNoticeBar/meta/NoticeBar.stories.mdx:15]:
```suggestion
  template: `<VNoticeBar>{{ args.text }}</VNoticeBar>`,
  components: { VNoticeBar },
```
Can we rename the component and the file here?

Comment (dhruvkb) [reply src/components/VNoticeBar/meta/NoticeBar.stories.mdx]: Good catch! Sure we can fix this. Made the change in 360a6e1.

**Review by obulat — APPROVED** [review]
Nice that the fix was this simple!
I just spent a frustratingly long amount of time trying to understand why I can see a "Controls/Actions" panel at the bottom of the "Canvas" tab in the deployed storybook, and cannot see it in the local version (I could only see the Controls in the "Docs" tab). Tried all sort of code changes, nothing worked.
It turns out that the panel was always there, it was just hidden. Pressing <kbd>A</kbd> and <kbd>D</kbd> a couple of times fixed it 🤯
I've opened another PR on top of this one to fix the ErrorSection.

**Comment by dhruvkb** [comment]
@obulat a fix for the hidden panel is supposed to ship with Storybook v6.5 (currently in alpha).

**Review by sarayourfriend — APPROVED** [review]
LGTM! Also part of #1220.

**Pull request merged**